.data/vSMR_WIP_areas.txt

```diff
@@ -12,25 +12,31 @@ COORD:N052.12.22.287:E000.10.20.976
 COORD:N052.12.28.895:E000.10.28.275
 COORD:N052.12.31.207:E000.10.30.183
 
-EGPH M (Q-S) - NOTAM A2226/25 ends 2025-07-02 2300
-COORD:N055.56.59.105:W003.21.12.808
-COORD:N055.56.54.809:W003.20.58.729
-COORD:N055.56.53.421:W003.20.59.745
-COORD:N055.56.57.790:W003.21.13.948
+EGLL N1 - NOTAM A4416/25 ends 2025-09-30 0430
+COORD:N051.27.57.403:W000.26.02.599
+COORD:N051.27.57.393:W000.26.05.023
+COORD:N051.28.01.250:W000.26.05.068
+COORD:N051.28.01.288:W000.26.02.644
```

Comment:
I think, given the NOTAM is only active 1845z-0430z (overnight) - there's not much point us including this on the WIP display.